tg/models.py

```diff
@@ -244,17 +245,13 @@ def _load_next_chats(self) -> None:
             log.error(f"get chat ids error: {result.error_info}")
             return None
 
-        chats = result.update["chat_ids"]
-        if not chats:
+        chat_ids = result.update["chat_ids"]
+        if not chat_ids:
             self.have_full_chat_list = True
-            return chats
+            return
 
-        for chat_id in chats:
-            # TODO: fix this, we shouldn't have any duplicates
-            if chat_id not in self.chat_ids:
-                self.chat_ids.append(chat_id)
-                chat = self.fetch_chat(chat_id)
-                self.chats.append(chat)
+        for chat_id in chat_ids:
+            self.fetch_chat(chat_id)
```

Comment:
I think I found error. Adding
            chat = self.fetch_chat(chat_id)
            self.add_chat(chat)
Resolved the issue